include/CL/sycl/device.hpp

```diff
@@ -27,6 +27,16 @@
 #include "CL/sycl/device_selector.hpp"
 #include "CL/sycl/platform.hpp"
 
+#ifndef WEAK_ATTRIB_PREFIX
```

Comment:
I thought of that too, but `include/CL/sycl/detail/global_config.hpp` contained strictly behavioral defines and no platform specific ones, that's why I was reluctant to put it there. But yeah, it might house platform specific stuff as well.